src/web/HttpSession.hpp

```diff
@@ -51,13 +52,15 @@ template <SomeServerHandler HandlerType>
 class HttpSession : public impl::HttpBase<HttpSession, HandlerType>,
                     public std::enable_shared_from_this<HttpSession<HandlerType>> {
     boost::beast::tcp_stream stream_;
+    util::Config config_;
```

Comment:
Let me know if reference is preferred. 👉

Reply:
Why not just the limit for the queue size?

Reply:
I think maybe in future there will be more config?

Reply:
This code will be deleted soon, so there will be not many changes here.